internal/metastore/model/collection.go

```diff
@@ -0,0 +1,25 @@
+package model
+
+import (
+	"github.com/milvus-io/milvus/internal/common"
+	"github.com/milvus-io/milvus/internal/proto/commonpb"
+)
+
+type Collection struct {
+	TenantID             string
```

Comment:
I was wondering what is the reason we defined model again but not use pb directly?

Reply:
[etcd Pb ](https://github.com/milvus-io/milvus/blob/015a2f08663c98b94adf2fba161d1c8c9bb4e1ba/internal/proto/etcdpb/etcd_meta.pb.go#L198) should only use within  etcd kv metastore, we need a high-level abstract model that can be used to other storage types of metastore.